geopandas/geodataframe.py

```diff
@@ -1401,18 +1401,29 @@ def merge_geometries(block):
         return aggregated
 
     # overrides the pandas native explode method to break up features geometrically
-    def explode(self, column=None, **kwargs):
+    def explode(self, column=None, ignore_index=False, add_multiindex=True):
         """
         Explode muti-part geometries into multiple single geometries.
 
         Each row containing a multi-part geometry will be split into
         multiple rows with single geometries, thereby increasing the vertical
         size of the GeoDataFrame.
 
-        The index of the input geodataframe is no longer unique and is
-        replaced with a multi-index (original index with additional level
-        indicating the multiple geometries: a new zero-based index for each
-        single part geometry per multi-part geometry).
+
+        Parameters
+        ----------
+        column : string, default None
+            If None, MultiGeometries in geometry column are converted to Single ones.
+            Else, explode another column following pandas behaviour.
```

Comment:
```suggestion
            Column to explode.
            In the case of GeoSeries, multi-part in geometries are converted to single-part.
            If None, an active geometry column is used.
```

Maybe better to have it more explicit like this?